src/MongoDB/Monitoring/CommandFailedEvent.c

```diff
@@ -195,10 +184,6 @@ static void php_phongo_commandfailedevent_free_object(zend_object* object)
 		zval_ptr_dtor(&intern->z_error);
 	}
 
-	if (!Z_ISUNDEF(intern->manager)) {
-		zval_ptr_dtor(&intern->manager);
```

Comment:
You can also remove the `manager` field from the corresponding struct as it's no longer used. Same applies to the other command event structs.